infra/testing.pinning

```diff
@@ -0,0 +1,7 @@
+Package: *
+Pin: release a=buster
+Pin-Priority: 700
+
+Package: *
```

Comment:
What if we pinned just `git` to testing?

Reply:
Could we get rid of Pin-Priority if we do that?

Reply:
I don't think so, if you don't set priority, packages may end up being installed from testing instead of buster (that's what happenned to me initially as i used `stable` instead of `buster`, and wrecking the setup of mercurial).

+ it's only 3Kb more in your image :smirk:

Reply:
If we do ` Package: git`, don't we make it sure that only git can be installed from testing?